shared/chat/conversation/info-panel/index.js.flow

```diff
@@ -1,6 +1,14 @@
 // @flow
 import {Component} from 'react'
 
+export type MuteRowProps = {
```

Comment:
if this is going to be a shared component (aka a plain .js file) we don't need a sep. flow file and its likely more dangerous to do so as they can drift out of sync

Reply:
Done